samples/bluetooth/peripheral_hids_keyboard/README.rst

```diff
@@ -149,7 +155,7 @@ To test with `nRF Connect for Desktop`_, complete the following steps:
    The first one has the value ``02000F0000000000``, the second has the value ``0200000000000000``.
    These values correspond to press and release of character "l" with the Shift key pressed.
 #. In nRF Connect, select the HID Report (which has UUID 0x2A4D and the properties Read, WriteWithoutResponse, and Write).
-   Enter ``02`` in the text box and click the :guilabel:`Write` button.
+   Enter ``02`` in the text box and click the :guilabel:`tick mark` button.
```

Comment:
tick mark ? I don't understand what it means in this context

Reply:
A comment about this in https://projecttools.nordicsemi.no/jira/browse/NCSDK-8094